corehq/apps/export/forms.py

```diff
@@ -405,11 +406,12 @@ def get_edit_url(self, export):
         return reverse(EditNewCustomFormExportView.urlname,
                        args=(self.domain_object.name, export._id))
 
-    def get_form_filter(self):
+    def get_form_filter(self, mobile_user_and_group_slugs):
         return filter(None, [
             self._get_datespan_filter(),
-            self._get_group_filter(),
-            self._get_user_filter()
+            self._get_group_filter(mobile_user_and_group_slugs),
+            self._get_user_filter(mobile_user_and_group_slugs),
+            self._get_user_ids_filter(mobile_user_and_group_slugs)
```

Comment:
Location are just shown in the views as of now. Filtering on their basis is yet to be implemented.

Just making less changes and keeping things close to how they were. Will look for improvements and breaking the architecture next.